lib/posts.ts

```diff
@@ -169,3 +168,14 @@ export type Post = Database['public']['Tables']['posts']['Row'] & {
     published: boolean;
   }[];
 };
+
+export async function updatePost(
+  id: number,
+  payload: Partial<Omit<Post, 'id'>>,
+) {
+  return await supabase.from('posts').update(payload).eq('id', id);
+}
+
+export function toSlug(title: string) {
+  return title.replaceAll(' ', '-').toLowerCase();
```

Comment:
```suggestion
  return title.trim().replaceAll(' ', '-').toLowerCase();
```

I would trim the both start & end to be sure to avoid --hey-there- by error.

Reply:
Sounds good, but I would trim the title too :). Will commit it